### ISSUE TITLE
for source highlighting (after <obj>??), %colors neutral is the same as %colors lightbg

### ISSUE THREAD

**Issue by anntzer** [ipython/ipython#9719]
A picture is worth a thousand words, they say...
![screenshot_20160707_192209](https://cloud.githubusercontent.com/assets/1322974/16675678/231db1dc-4478-11e6-92e9-87b8a5407bea.png)

**Comment by Carreau** [comment]
Yes, the things that color sources for oinspect does not use pygments and is identical to lightbg one. 
We should switch it to pygments,  or tweak the theme a bit.
